backend/src/utils/log_publisher.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import asyncio
+from enum import Enum
+from src.websockets.types import Message, MessageTypes
+from src.websockets.connection_manager import connection_manager
+
+log = logging.getLogger(__name__)
+
+class LogPrefix(Enum):
+    USER = "USER"
+
+async def publish_log(prefix: LogPrefix, msg: str, loglevel: int, name: str):
```

Comment:
The log level refers to how serious the log is. So the most serious log levels are CRITICAL then ERROR where as most of our logging is at the INFO level. It's use is to establish a hierarchy to enable you to filter logs, for example if something is going wrong in the app it might make sense to only look at WARNING CRITICAL and ERROR logs first because there's a good chance the log will be in one of those categories as opposed to info logs which we send for just general information.
If that explanation doesn't make a lot of sense then there is a section on log levels in the [python docs](https://docs.python.org/3/library/logging.html#logging-levels) that might be worth a read :)

Reply:
Thanks very much, makes sense :)